LokiLabs/microsoft-graph-explorer-v4
# AB#38584 Toggle application mode sample queries

## Overview

This PR allows switching between the different lists of sample queries used between user delegated mode and application mode. 

### Demo

![Animation](https://user-images.githubusercontent.com/47064505/126561187-cf0c64f3-2651-4e20-b955-93710e56da06.gif)

## Testing Instructions

* Checkout to branch 
* Run `npm start`
* Run using http://localhost:3000/?devx-api=https://gi21devxapi-devtest.azurewebsites.net&org=LokiLabs&branchName=ADD_YOUR_BRANCH_NAME_HERE

NOTE: ADD_YOUR_BRANCH_NAME_HERE should be a branch based on the devx_content repo, if you dont have a branch or didnt create one. You can use '38584-app-sample-queries'.
